packages/react-components/src/components/AppFrame/stories-helpers.tsx

```diff
@@ -0,0 +1,240 @@
+import * as React from 'react';
+
+import * as Icons from '@livechat/design-system-icons';
+
+import { useAppFrame } from '../../providers/AppFrameProvider';
+import { Badge } from '../Badge';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { Tag } from '../Tag';
+import { Heading, Text } from '../Typography';
+
+import { SubNavBar, SubNavBarList, SubNavBarListItem } from './components';
+interface ExampleAppContentProps {
+  showToggle: boolean;
+  topBarVisible: boolean;
+  handleTopBarButtonClick: () => void;
+}
+
+export const ExampleAppContent: React.FC<ExampleAppContentProps> = ({
+  showToggle,
+  topBarVisible,
+  handleTopBarButtonClick,
+}) => {
+  const { isSubNavBarVisible, toggleSubNavBarVisibility } = useAppFrame();
+
+  return (
+    <div
+      style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        flexDirection: 'column',
+      }}
+    >
+      <Heading size="lg">App content</Heading>
+      {showToggle && (
+        <>
+          <Text>Set sub-navigation visibility</Text>
+          <Button onClick={toggleSubNavBarVisibility}>
+            {isSubNavBarVisible ? 'Visible' : 'Hidden'}
+          </Button>
+        </>
+      )}
+      <>
+        <Text>Set top-bar visibility</Text>
+        <Button onClick={handleTopBarButtonClick}>
+          {topBarVisible ? 'Close' : 'Open'}
+        </Button>
+      </>
+    </div>
+  );
+};
+
+export const getChatsMenu = (
+  activeSubItem: number,
+  handler: (o: number) => void
+) => (
+  <SubNavBar noGaps title="Simple list">
```

Comment:
I'd suggest naming it `SideNavigation` and `SideNavigationItem` as these naming patterns are already used in the AA application.